website/www/site/assets/icons/extensive-icon.svg

```diff
@@ -0,0 +1,7 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="112" height="112" fill="none" viewBox="0 0 112 112">
+    <path fill="#FFE5D9" d="M56 112c30.928 0 56-25.072 56-56S86.928 0 56 0 0 25.072 0 56s25.072 56 56 56z"/>
+    <path fill="#FBC23E" d="M78.354 38.556v-7.83h-7.537C70.744 27.553 68.252 25 65.177 25c-3.07 0-5.568 2.553-5.64 5.726H52v7.83c3.118 0 5.646 2.63 5.646 5.871 0 3.242-2.528 5.871-5.646 5.871v7.83h7.53c0 3.243 2.529 5.872 5.647 5.872 3.118 0 5.646-2.63 5.646-5.871h7.53v-7.83c3.119 0 5.647-2.63 5.647-5.872s-2.522-5.87-5.646-5.87z"/>
+    <path fill="#FF570B" d="M71.084 83.584l7.496.027-.064-7.305c3.041-.061 5.46-2.466 5.437-5.441-.024-2.976-2.49-5.401-5.53-5.49l-.064-7.304-7.495-.027c.025 3.024-2.469 5.46-5.576 5.453-3.106-.013-5.639-2.472-5.67-5.498l-7.496-.027.066 7.3c-3.106-.014-5.602 2.428-5.576 5.452.025 3.024 2.564 5.484 5.67 5.497l.065 7.299 7.496.027c.025 3.024 2.564 5.484 5.67 5.497 3.101.006 5.597-2.436 5.571-5.46z"/>
+    <path fill="#fff" d="M52.177 38.556v-7.83h-7.772C44.33 27.553 41.754 25 38.59 25c-3.166 0-5.742 2.553-5.817 5.726H25v7.83c3.215 0 5.823 2.63 5.823 5.871 0 3.242-2.608 5.871-5.823 5.871v7.83h7.766c0 3.243 2.607 5.872 5.823 5.872 3.215 0 5.822-2.63 5.822-5.871h7.766v-7.83c3.216 0 5.823-2.63 5.823-5.872s-2.607-5.87-5.823-5.87z"/>
+    <path stroke="#000" stroke-linecap="round" stroke-width="3" d="M84 54l5-3M84.405 62.473l4.192 4.053M86 58.306l7.423.394"/>
+</svg>
```

Comment:
Is there a reason these files are in `assets/icons`? It looks like the rest of our images are in [`static/images`](https://github.com/apache/beam/tree/website-revamp/website/www/site/static/images)

Reply:
Yes, `static/images` is used to store png/jpeg images. And `assets/icons` is recommended to place the svg icons. Since the icons in `assets/icons` can be processed by Hugo Pipes. Thus, instead of using `<img src="./images/extensive-icon.svg`, I can use `{{ $icon := resources.Get "icon/extensive-icon.svg" }}`. With that way, I can easily access to svg code and modify it by playing around the `$icon`.

Reply:
Ah neat, thanks for the explanation